### ISSUE TITLE
Error Message When Accessing File Permissions Page

### ISSUE THREAD

**Issue by posixeleni** [IQSS/dataverse#2624]
User received an email that someone requested access to their file(s) from this dataset https://dataverse.harvard.edu/dataset.xhtml?persistentId=hdl:1902.1/16786 but when they click on the link in the email they are taken to a "500 Internal Server Error" page.

https://dataverse.harvard.edu/permissions-manage-files.xhtml?id=65402

I tried accessing the File Permissions page for this same dataset from the UI and am receiving the same error.

May be related to #2602

**Comment by sekmiller** [comment]
This is a duplicate of #2602

**Comment by pdurbin** [comment]
This is from https://help.hmdc.harvard.edu/Ticket/Display.html?id=228111

**Comment by sbarbosadataverse** [comment]
@sekmiller 
This is not the same issue as 2602 where a user is taken to a dataset, no error message, just the wrong dataset

**Comment by pdurbin** [comment]
@sbarbosadataverse and I emailed the user to see if the problem is still happening.

**Comment by pdurbin** [comment]
The user wrote back to @sbarbosadataverse and said, "Yes, it looks like this has been resolved - I was just able to approve a request for access."

I just went through the whole workflow (for the first time) and it seems to work fine. In my scenario, Fiona Finch requests access to "trees.png" from Sabrina Spruce. Screenshots tell the story:

![4 _mailinator_-_free _disposable _email _-_2015-11-09_12 14 09](https://cloud.githubusercontent.com/assets/21006/11040665/6b4ca144-86db-11e5-9bd4-4d8848613b34.png)

![account_-_root_dataverse_-_2015-11-09_12 08 15](https://cloud.githubusercontent.com/assets/21006/11040599/26c47664-86db-11e5-9885-8fc736d0fc1e.png)

![file_permissions_-_spruce_goose_-_2015-11-09_12 08 51](https://cloud.githubusercontent.com/assets/21006/11040600/26c835f6-86db-11e5-9497-f9fbf1c6c774.png)

![file_permissions_-_spruce_goose_-_2015-11-09_12 09 11](https://cloud.githubusercontent.com/assets/21006/11040598/26c2119e-86db-11e5-9850-491e48e8e137.png)

![5 _mailinator_-_free _disposable _email _-_2015-11-09_12 10 17](https://cloud.githubusercontent.com/assets/21006/11040601/26cf903a-86db-11e5-9892-049c833f34d8.png)

The bottom line for me is that the link seems to go to the right place. Sabrina spruce gets a notification and an email to manage permissions on trees.png, which is dataset 10 for me ("Spruce Goose" per below): http://localhost:8080/permissions-manage-files.xhtml?id=10

(Pro tip from #2110 on how to get a dataset title based on an id:

curl -s http://localhost:8080/api/datasets/10 | jq '.data.latestVersion.metadataBlocks.citation.fields[] | select(.typeName=="title").value'

"Spruce Goose")

I don't see any dev work to be done here. Passing to QA.

**Comment by pdurbin** [comment]
> I don't see any dev work to be done here.

Actually, I did make a minor change to increase logging to help me debug this issue: 763ed2e

You won't see this in the logs by default because I set it to the "FINE" level. Here's how to see it:

asadmin set-log-levels edu.harvard.iq.dataverse.UserNotificationServiceBean=FINE

asadmin set-log-levels edu.harvard.iq.dataverse.MailServiceBean=FINE

(I recently documented this at http://guides.dataverse.org/en/4.2.2/developers/debugging.html .)

**Comment by kcondon** [comment]
@pdurbin This functionality was tested fully in v4.2.1. It seems you are asking me to test the logging levels, right?

**Comment by pdurbin** [comment]
@kcondon nah. You don't need to test the logging unless you really want to.

This issue made it into my queue because @sbarbosadataverse identified it as critical. Again, the RT ticket is resolved. I couldn't reproduce the bug. And the related issue #2602 has been closed. I'm all for closing issues we can't reproduce them.

**Comment by kcondon** [comment]
Retested notification workflow and it works. Closing.
